src/main/java/uk/gov/pay/directdebit/common/model/ErrorResponse.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.pay.directdebit.common.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@JsonInclude(Include.NON_NULL)
+public class ErrorResponse {
+    
+    @JsonProperty("error_identifier")
+    private final ErrorIdentifier identifier;
+    
+    @JsonProperty("message")
+    private final List<String> messages;
+    
+    @JsonProperty("reason")
+    private final String reason;
```

Comment:
So we can have multiple messages but only one reason?

Reply:
Hmm this was copied over from connector actually, I think we can dispense with the reason for Direct Debit Connector as it's not used. I wasn't planning on making this model common between micro-services, but should we?